src/component/BookActions.js

```diff
@@ -0,0 +1,49 @@
+import { useDispatch } from 'react-redux';
+import { useState } from 'react';
+import { addBook } from '../redux/books/books';
+
+const BookActions = () => {
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+  const dispatch = useDispatch();
+
+  return (
+    <form>
+      <h2 className="addbook">ADD NEW BOOK</h2>
+      <input
+        type="text"
+        placeholder="Book Title"
+        onChange={(event) => {
+          setTitle(event.target.value);
+        }}
+      />
+      <input
+        type="text"
+        placeholder="Author"
+        onChange={(event) => {
+          setAuthor(event.target.value);
+        }}
+      />
+      <button
+        onClick={(event) => {
+          event.preventDefault();
+          dispatch(
+            addBook({
+              id: Math.floor(
+                Math.random() * Math.floor(Math.random() * Date.now())
+              ),
+              author,
+              title,
+            })
+          );
+        }}
```

Comment:
- [ ] I think it is very important to follow the design here when creating the form so please make the form the same as the Zeplin design here and make sure to add the placeholders here let the user know what he/she is entering 💯🔥🚀✨

- Your design
![image](https://user-images.githubusercontent.com/58553711/204062898-d2324440-c5bf-44bc-8c4f-e99fba11bf44.png)

- given design
![image](https://user-images.githubusercontent.com/58553711/204062919-aba12a53-901f-44b9-8131-4e759eed2f53.png)

Reply:
right but i am not using a category so I will make the two inputs bigger

Reply:
and fixed